test/nuxeo-date-test.html

```diff
@@ -0,0 +1,91 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+<head>
+  <title>nuxeo-date tests</title>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <script>
+    // force use of polyfill: see ELEMENTS-56
+    window.fetch = null;
+  </script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../nuxeo-elements/test/test-helpers.js"></script>
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../widgets/nuxeo-date.html">
+  <link rel="import" href="../widgets/nuxeo-tooltip.html">
+  <script src="../../moment/moment.js"></script>
+</head>
+<body>
+
+  <test-fixture id="equalFormat">
+    <template>
+      <nuxeo-date format="LLL" tooltipFormat="LLL"></nuxeo-date>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="globalconfigTooltipFormat">
+    <template>
+      <nuxeo-date format="LLL"></nuxeo-date>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="differentFormat">
+    <template>
+      <nuxeo-date tformat="relative" tooltipFormat="LLL"></nuxeo-date>
+    </template>
+  </test-fixture>
+
+
+  <script>
+    function setNuxeoConfigDateFormat(format, value) {
+      window.Nuxeo = window.Nuxeo || {};
+      window.Nuxeo.UI = window.Nuxeo.UI || {};
+      window.Nuxeo.UI.config = window.Nuxeo.UI.config || {};
+      window.Nuxeo.UI.config[format] = window.Nuxeo.UI.config[format] || value;
+    }
+    /* eslint-disable no-unused-expressions */
+
+    suite('<nuxeo-date>', () => {
+      let element;
+
+      test('Should hide the nuxeo-tooltip if provided format and tooltipFormat are equal', () => {
+        element = fixture('equalFormat');
+        element.date = moment();
```

Comment:
If we are defining this after the fixture, I guess we can reuse the same fixture for everything and then set the properties 🤷‍♂️ 
Also, when we use this kind of approach, _flush_ mechanism is usually needed, otherwise we might end not validating what we want.

Reply:
yes initially I thought of it but for some reason it wasn't working. Now it is, much more straightforward :)